### ISSUE TITLE
Docker update problem

### ISSUE THREAD

**Issue by marithpl** [marthoc/docker-deconz#370]
I'm using a Deconz 2.09.01, I wish to update to 2.10.2 but I have some problems.
I've updated a docker image, run a docker container and after that I have to create a new Conbee II password.
When I connect to VNC I see all devices as hex (like 000x0 or sth like that).

What is the problem and what is the best way to make an update.

Deconz works on Synology DSM 7.0 Docker Container

**Comment by samsam-rolon** [comment]
Hello
I got the same issue with "ghost" devices whitch was too long disconnected from my network. I think this is an issue to the deConz software on https://github.com/dresden-elektronik/deconz-rest-plugin and not related to docker.

**Comment by marithpl** [comment]
So you can join to my request:
https://github.com/dresden-elektronik/deconz-rest-plugin/issues/4561

**Comment by Mimiix** [comment]
@marithpl As it is specific to the docker version and not deCONZ itself.

**Comment by ict** [comment]
I experienced the same issue when watchtower updated my deCONZ container last night.
The problem might be that the documentation recommends to host-mount or volume-mount `/opt/deconz` to save the configuration, when in fact the configuration seems to be stored in `/root/.local/share/dresden-elektronik/deCONZ`, which is also referenced in the Dockerfile. I have adapted my docker config for this, let's see if it helps.

Restoring a backup does work, too. But it requires manual effort.
